src/Client.php

```diff
@@ -107,6 +107,40 @@ public function ezValue($stat, $value)
     }
 
     /**
+     * List stat alerts using the Alerts API. Used to look up alert IDs of alerts assigned
+     * to certain stats.
+     *
+     * @see https://www.stathat.com/manual/alerts_api
+     * @throws \Exception
+     */
+    public function listAlerts()
+    {
+        if (! isset($this->config['access_token'])) {
+            throw new Exception('StatHat Alerts API Access Token not set.');
+        }
+
+        $result = $this->deleteAlertCurl('x/'.$this->config['access_token'].'/alerts/'.$alert_id);
+
+        return $result;
+    }
+
+    /**
+     * Get listing of all stat alerts using the Alerts API.
+     *
+     * @see https://www.stathat.com/manual/alerts_api
+     * @throws \Exception
+     */
+    public function getAlerts()
+    {
+        if (! isset($this->config['access_token'])) {
+            throw new Exception('StatHat Alerts API Access Token not set.');
+        }
```

Comment:
Also this is super nit-picky (sorry!) but could you add a space between the conditional and the request line here (like in the method above)? 😇